docs/spec_draft.md

```diff
@@ -838,6 +839,74 @@ operation.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.pad
+
+### Semantics
+
+Expands `operand` by padding around the array as well as between the elements of
+the array with the given `padding_value`.
+
+`edge_padding_low` and `edge_padding_high` specifies the amount of padding added
+at the low-end (next to index 0) and the high-end (next to the highest index) of
+each dimension respectively. The amount of padding can be negative, where the
+absolute value of negative padding indicates the number of elements to remove
+from the specified dimension.
+
+`interior_padding` specifies the amount of padding added between any two
+elements in each dimension which may not be negative. Interior padding occurs
+before edge padding such that negative edge padding will remove elements from
+the interior-padded operand.
+
+### Operands
+
+| Name                | Type                               |
+|---------------------|------------------------------------|
+| `operand`           | tensor of any supported types      |
+| `padding_value`     | tensor of any supported types      |
```

Comment:
padding is a scalar value?

Reply:
Padding value is indeed scalar, but the ODS accepts HLOTensor, and the verifier checks that this is rank 0. This is the case with https://github.com/tensorflow/tensorflow/blob/master/tensorflow/compiler/xla/service/shape_inference.cc#L566-L569 and https://github.com/openxla/stablehlo/blob/main/stablehlo/dialect/StablehloOps.cpp#L4451-L4456